### PR TITLE
Add basic Makefile

### PR DESCRIPTION
**Pull request by maxfisher-g**
Begins work for #400, but still can be expanded significantly.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+REGISTRY=gcr.io/ossf-malware-analysis
+
+.PHONY: all all_docker analysis_docker
+
+all: all_docker
+
+all_docker:
```
Comment (oliverchang) [inline Makefile:8]: Does this need a dependency on analysis_docker?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	bash build/build_docker.sh
+
+analysis_docker:
+	docker build -t ${REGISTRY}/analysis -f cmd/analyze/Dockerfile .
```
Comment (oliverchang) [inline Makefile:12]: Any reason to delete the bash script and replace things inline here? May be better to be consistent with the other docker build scripts here.

File: Makefile (hunk 1 above)
Comment (maxfisher-g) [reply Makefile:8]: I was just using it as a default 'all'. `all_docker` also builds analysis docker image.

File: Makefile (hunk 2 above)
Comment (maxfisher-g) [reply Makefile:12]: From what I understood talking to @calebbrown, we actually wanted to move towards everything being in the Makefile. That way it's a central point for dev tasks. I just left the build_docker.sh script there for now, so we can at least start with something in the Makefile. But if this is too incremental of a change currently, I can move the `build_docker` script into here.

Comment (oliverchang) [reply Makefile:12]: Got it, this SGTM then.

File: Makefile (hunk 1 above)
Comment (oliverchang) [reply Makefile:8]: Ah right, the bash script also builds it. 

Can you just add a comment here explaining the distinction between the two cases? i.e. build_docker.sh builds everything, including analysis.

One other option is to make the build_docker.sh take in arguments, and the Makefile can just pass arguments there to build the relevant images.

Comment (maxfisher-g) [reply Makefile:8]: Since I'm hoping to merge `build_docker.sh` into this script, I'll probably just leave a comment here for now

**Comment by maxfisher-g** [comment]
The shellcheck action is broken, I will fix it in another PR by making it run the check_scripts rule in this Makefile

**Review by oliverchang — APPROVED** [review]

**Review by oliverchang — APPROVED** [review]

**Pull request merged**